test/index.test.js

```diff
@@ -0,0 +1,97 @@
+describe('Web test', () => {
+  let createServer
+  let server
+  let mockClaimRepository
+
+  beforeAll(async () => {
+    jest.mock('../server/repository/claim-repository')
+    jest.mock('../server/repository/minetype-repository')
+    jest.mock('../server/services/message-service')
+    createServer = require('../server')
+    mockClaimRepository = require('../server/repository/claim-repository')
+  })
+
+  beforeEach(async () => {
+    server = await createServer()
+    await server.initialize()

```

Comment:
I think to make sure that each server is a fresh instance for each request, it needs to be done like this.

Reply:
I was thinking that we don't necessarily need a fresh server instance for each test because we mock anything that would store state. Then again, I don't know the inner workings of Hapi and whether it will carry over any state between calls so it probably is safer to reset as you are doing.